backend/apps/volontulo/serializers.py

```diff
@@ -11,12 +11,34 @@
 from apps.volontulo import models
 
 
+class OrganizationSerializer(serializers.HyperlinkedModelSerializer):
+
+    """REST API organizations serializer."""
+
+    slug = serializers.SerializerMethodField()
+
+    class Meta:
+        model = models.Organization
+        fields = (
+            'id',
+            'name',
+            'slug',
+            'url',
+        )
+
+    @staticmethod
+    def get_slug(obj):
+        """Returns slugified name."""
+        return slugify(obj.name)
+
+
 class OfferSerializer(serializers.HyperlinkedModelSerializer):
 
     """REST API offers serializer."""
 
     slug = serializers.SerializerMethodField()
     image = serializers.SerializerMethodField()
+    organization = OrganizationSerializer(many=False, read_only=True)
```

Comment:
Will there be a join by default? Or we will be lazily hitting the db with N additional selects to get the organizations?

Reply:
I am not quite sure, but that is the solution I have found for nested items. If you have better solution, just let me know.